OFEP-provider-metadata-feature-discovery.md

```diff
@@ -0,0 +1,36 @@
+## OFEP-provider-metadata-feature-discovery
+
+## State: DRAFTING
+
+This OFEP proposes a solution for "feature discovery" in the context of providers, in order to signal to the SDK and `application authors` what features are available in a particular provider.
```

Comment:
"feature" is quite an overloaded word in the domain of open feature. For the sake of sanity, could we use "capability" instead? We'd use capability discovery to discover which capabilities a given provider has. Language like "does this provider support the provider events capability" seems pretty natural to me.

Reply:
Great point, love it.